bin/travis/runbehat.sh

```diff
@@ -2,4 +2,4 @@
 
 cd $HOME/build/ezplatform
 
-php bin/behat -vv --profile=platformui --tags='~@edge'
+php bin/behat -vv --profile=platformui --tags='~@edge && ~@notWorking'
```

Comment:
I think we already have a tag called \@broken, should use the same so meta repo does not need updates